packages/babel-preset-typescript/src/index.ts

```diff
@@ -65,7 +66,13 @@ export default declarePreset((api, opts: Options) => {
           {
             test: /\.cts$/,
             sourceType: "script",
-            plugins: [[transformTypeScript, pluginOptions(false, true)]],
+            parserOpts: {
+              allowImportExportEverywhere: true,
+            },
+            plugins: [
+              [transformModulesCommonJS, { allowTopLevelThis: true }],
+              [transformTypeScript, pluginOptions(false, true)],
+            ],
```

Comment:
This is probably not good, since it means incompatibility with `allExtensions`. I want to know your opinion.

Reply:
`babel.config.cts` requires it, and almost all `cts` files require this. Because when using `require` instead of `import`, tsc cannot get type information, even for built-in modules like `fs`.